### ISSUE TITLE
Model inheritance and translation

### ISSUE THREAD

**Issue by Remiz** [deschler/django-modeltranslation#308]
Hello, I noticed that my models with translation using inheritance were broken with the upgrade to Django 1.8.

I setup a minimal test case on a new project and find a similar issue.

```python
# models.py
class BaseQuestion(models.Model):
    label = models.CharField(max_length=400, null=True, blank=True)  # Translated
    position = models.IntegerField(default=0)


class QuestionScale(BaseQuestion):
    low_anchor = models.CharField(max_length=100, null=True, blank=True)  # Translated


class QuestionMultiChoice(BaseQuestion):
    multiple_answer = models.BooleanField(default=False) 
    # Nothing is translated for this class

# translation.py
class BaseQuestionTranslationOptions(TranslationOptions):
    fields = ('label',)

translator.register(BaseQuestion, BaseQuestionTranslationOptions)


class QuestionScaleTranslationOptions(TranslationOptions):
    fields = ('low_anchor',)

translator.register(QuestionScale, QuestionScaleTranslationOptions)
```

So when I open my shell and type
```python
>>>question_scale = QuestionScale(label='This is a test')
>>>question_scale.save()
>>>question_scale.label_en
'This is a test'
>>>question_multichoice = QuestionMultiChoice(label='This is also a test')
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/Users/remi/.virtualenvs/debug/lib/python2.7/site-packages/modeltranslation/translator.py", line 213, in new_init
    old_init(self, *args, **kwargs)
  File "/Users/remi/.virtualenvs/debug/lib/python2.7/site-packages/django/db/models/base.py", line 480, in __init__
    raise TypeError("'%s' is an invalid keyword argument for this function" % list(kwargs)[0])
TypeError: 'label_en' is an invalid keyword argument for this function
```

So maybe it comes from the way I declare my translations, maybe QuestionMultiChoice is supposed to be declared in there (which field should I put? Since I don't have any translation for that child class...).

Thanks.

**Comment by Kniyl** [comment]
Try registering your QuestionMultiChoice with no added translation fields:
```python
class QMCTranslationOptions(TranslationOptions):
    fields = ()

translator.register(QuestionMultiChoice, QMCTranslationOptions)
```
so that model translation will force rebuilding QuestionMultiChoice's fields cache.

You might encounter #304 in doing so, if it occurs, try changing line 153 of modeltranslation/translator.py into
```python
    model._meta._expire_cache()
    model._meta.get_fields()
```
works for me with minimal testing.

**Comment by Remiz** [comment]
Thanks. I added the empty translation declaration to my model, plus the new version fixed the issue #304.
